src/host/ft_host/InitTests.cpp

```diff
@@ -238,7 +238,6 @@ MODULE_SETUP(ModuleSetup)
     VERIFY_WIN32_BOOL_SUCCEEDED_RETURN(FreeConsole());
 
     // Wait a moment for the driver to be ready after freeing to attach.
-    Sleep(1000);

```

Comment:
I ran these feature tests probably hundreds of times without exaggerating. With that sleep in place the tests take about a minute to complete. With it removed it takes 5 seconds. Initially I kept this sleep, but in my last commit I was curious to see if the tests also complete fine without the sleep on the CI. Since it seemed like it did, I felt like this might be a good related addition to this PR, as it caused me a ton of pain during testing.

Reply:
I mean, I won't complain about CI finishing faster haha